src/js/components/Drop/DropContainer.js

```diff
@@ -166,32 +166,20 @@ const DropContainer = forwardRef(
           let bottom;
           let maxHeight = containerRect.height;
           if (align.top) {
-            if (align.top === 'top') {
-              ({ top } = targetRect);
-            } else {
-              top = targetRect.bottom;
-            }
-
-            // Calculate visible area underneath the control w.r.t window height
-            const percentVisibleAreaBelow =
-              100 - (targetRect.bottom / windowHeight) * 100;
+            const alignToTop = align.top === 'top';
+            const switchTop = !alignToTop ? targetRect.top : targetRect.bottom;
```

Comment:
just opinion not sure about the naming for `switchTop`

Reply:
Agree the name is confusing. I changed it to `switchTarget` since this props is new target position the drop will be aligned to if we flip the drop.